sql/src/test/java/io/crate/integrationtests/PrivilegesIntegrationTest.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import com.google.common.collect.ImmutableSet;
+import io.crate.action.sql.Option;
+import io.crate.action.sql.SQLActionException;
+import io.crate.action.sql.SQLOperations;
+import io.crate.operation.user.User;
+import io.crate.settings.SharedSettings;
+import io.crate.testing.SQLResponse;
+import org.elasticsearch.common.settings.Settings;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.EnumSet;
+
+import static io.crate.testing.SQLTransportExecutor.DEFAULT_SOFT_LIMIT;
+import static org.hamcrest.core.Is.is;
+
+public class PrivilegesIntegrationTest extends SQLTransportIntegrationTest {
```

Comment:
pls move that to the users subproject (I suggest package `io.crate.integrationtests`)